### PR TITLE
 Cards for triboson resonance search

### PR DESCRIPTION
**Pull request by xdlyu**

### CODE REVIEW

**Comment by perrozzi** [comment]
we strongly recommend to upload only one representative card and then a script to produce the others, in order not to fill the repository with a large amount of files with the risk of slow down

**Comment by perrozzi** [comment]
hi, the request to upload only one representative card and then a script to produce the others still holds

**Comment by xdlyu** [comment]
Thanks for your reminding. Now I have added an example for a representative mass point and given the script to produce all the mass points (75 samples in total).

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Triboson search/Triboson_M_R/Triboson_M_R_run_card.dat [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+#*********************************************************************
+# Beam polarization from -100 (left-handed) to 100 (right-handed)    *
+#*********************************************************************
+     0.0     = polbeam1 ! beam polarization for beam 1
+     0.0     = polbeam2 ! beam polarization for beam 2
+#*********************************************************************
+# PDF CHOICE: this automatically fixes also alpha_s and its evol.    *
+#*********************************************************************
+     lhapdf    = pdlabel   ! PDF set                                     
+     263400    = lhaid     ! if pdlabel=lhapdf, this is the lhapdf number#*********************************************************************
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Triboson search/Triboson_M_R/Triboson_M_R_run_card.dat]: same here

**Comment by xdlyu** [comment]
We will do search with 2016 data. So we choose this earlier PDF.

**Comment by perrozzi** [comment]
if so, then move the cards to the pre2017 folder instead of 2017

File: bin/MadGraph5_aMCatNLO/cards/production/pre2017/13TeV/Triboson search/Triboson_M_R/Triboson_M_R_customizecards.dat [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#set param_card mass 9000024 <MASS>
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/pre2017/13TeV/Triboson search/Triboson_M_R/Triboson_M_R_customizecards.dat]: can you please remove the commented lines?

Comment (xdlyu) [reply bin/MadGraph5_aMCatNLO/cards/production/pre2017/13TeV/Triboson search/Triboson_M_R/Triboson_M_R_customizecards.dat]: OK，I removed them just now.

**Pull request merged**